buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidTarget.groovy

```diff
@@ -46,12 +47,33 @@ abstract class AndroidTarget extends JavaLibTarget {
 
         // Butterknife support
         generateR2 = project.plugins.hasPlugin('com.jakewharton.butterknife')
+
+        // SqlDelight support
+        if (project.plugins.hasPlugin('com.squareup.sqldelight')) {
+            File sqldelightDir = new File(project.projectDir, "build/okbuck/sqldelight")
+            if (!sqldelightDir.exists()) {
```

Comment:
// this dir is set as source set, if it doesn't exist, then we can't read it later